ci-operator/config/openshift/hypershift/openshift-hypershift-main__periodics.yaml

```diff
@@ -36,27 +36,17 @@ images:
       - registry.ci.openshift.org/openshift/release:golang-1.17
   to: hypershift
 releases:
-  4-11:
```

Comment:
Please leave this, it is used by the conformance workflow for the dump step because only the 4.11 cli does not strep managed fields of objects